package.json

```diff
@@ -114,7 +114,7 @@
     "axe-core": "^4.10.2",
     "downshift": "^9.0.8",
     "draft-js": "^0.10.5",
-    "draftail": "^2.0.0-rc.6",
+    "draftail": "^2.0.1",
```

Comment:
In addition to the Sass deprecation changes, I’ve also relaxed the downshift dependency to make sure we use the same version in Draftail and in Wagtail (#12540).